tests/Buildalyzer.Workspaces.Tests/ProjectAnalyzerExtensionsFixture.cs

```diff
@@ -59,7 +59,7 @@ public void CreatesCompilationOptions()
         }
 
         [TestCase(false, 1)]
-        [TestCase(true, 2)]
+        [TestCase(true, 3)]

```

Comment:
Essentially this is the change in behavior, you don't need to add child projects now when `addProjectReferences` is used. If the old behavior is still desired, perhaps we should change the parameter to an enum and support both variants?